src/components/dex/Pair.vue

```diff
@@ -58,19 +58,43 @@ export default {
       );
     },
 
-    quoteVolume() {
-      return this.$formatNumber(_.get(this.currentTickerData, 'quoteVolume', 0));
-    },
-
     tableData() {
       return this.filteredMarkets().map(({ quoteCurrency, marketName }) => {
-        const tradeHistory = _.get(this.$store.state.tradeHistory, marketName, {});
-        const hasTradeHistory = tradeHistory && tradeHistory.trades && tradeHistory.trades.length > 0;
-        const price = {
-          price: this.$formatTokenAmount(hasTradeHistory ? tradeHistory.close24Hour : 0),
-          priceConverted: this.$formatMoney((hasTradeHistory ? tradeHistory.close24Hour : 0) * this.baseCurrencyUnitPrice),
-        };
-        return { asset: quoteCurrency, price, volume: this.quoteVolume, '24H change': this.percentChangeAbsolute };
+        const tickerData = this.tickerDataByMarket[marketName];
+        let price;
+        let quoteVolume;
+        let percentChange;
```

Comment:
You can one-line this.
```
let price, quoteVolume, percentChange;
```

Reply:
Actually Lint says "expected variable declaration to be on a new line"; so linter wants it the way it is.